pwa/src/components/common/codeBlock/codeBlock.tsx

```diff
@@ -1,19 +1,27 @@
 import * as React from "react";
 import "./codeBlock.css";
+import Prism from "prismjs";
+import "../../../styles/prism.css";
+import 'prismjs/components/prism-json';
+import 'prismjs/components/prism-yaml';
 
 interface CodeBlockProps {
   code: any,
-  language: "json" | "xml"
+  language: "json" | "xml" | "css" | "html" | "yaml"
 }
 
 export const CodeBlock: React.FC<CodeBlockProps> = ({ code, language }) => {
 
-  return (
-    <pre className="codeBlock">
-      <code className={`codeBlock-code--${language}`}>
+  React.useEffect(() => {
+    Prism.highlightAll();
+  }, [Prism])
+
+  return (<>
+    <pre>
```

Comment:
Since we have a `pre` element wrapping the entire component, we do not need the empty wrapper (`<></>`).